### PR TITLE
Remove references

### PR DESCRIPTION
**Pull request by leodido**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

NONE

**What this PR does / why we need it**:

Removes some more unnecessary, legacy references to Falco in Sysdig.

**Does this PR introduce a user-facing change?**:

```release-note
update(docker): use HOST_ROOT environment variable rather than SYSDIG_HOST_ROOT
update: EF_DROP_FALCO is now EF_DROP_SIMPLE_CONS
update: puppet module had been renamed from "sysdig-falco" to "falco"
```

### CODE REVIEW

File: docker/dev/docker-entrypoint.sh [hunk 1]
```diff
@@ -16,19 +16,21 @@
 # limitations under the License.
 #
 
-#set -e
+# set -e
 
 # Set the SYSDIG_SKIP_LOAD variable to skip loading the sysdig kernel module
 
 if [[ -z "${SYSDIG_SKIP_LOAD}" ]]; then
```
Comment (nathan-b) [inline docker/dev/docker-entrypoint.sh]: This symbol still has SYSDIG in it

File: docker/rhel/docker-entrypoint.sh [hunk 2]
```diff
@@ -16,19 +16,21 @@
 # limitations under the License.
 #
 
-#set -e
+# set -e
 
 # Set the SYSDIG_SKIP_LOAD variable to skip loading the sysdig kernel module
 
 if [[ -z "${SYSDIG_SKIP_LOAD}" ]]; then
```
Comment (nathan-b) [inline docker/rhel/docker-entrypoint.sh]: SYSDIG -> FALCO

File: docker/stable/docker-entrypoint.sh [hunk 3]
```diff
@@ -16,19 +16,21 @@
 # limitations under the License.
 #
 
-#set -e
+# set -e
 
 # Set the SYSDIG_SKIP_LOAD variable to skip loading the sysdig kernel module
 
 if [[ -z "${SYSDIG_SKIP_LOAD}" ]]; then
```
Comment (nathan-b) [inline docker/stable/docker-entrypoint.sh]: SYSDIG -> FALCO

File: scripts/falco-probe-loader [hunk 4]
```diff
@@ -0,0 +1,408 @@
...
+			local -r kernel_version_patch=$(echo ${kernel_version} | cut -d. -f3)
+
+			if [ "${kernel_version_patch}" == "0" ]; then
+				kernel_version="${kernel_version_major}.${kernel_version_minor}"
+			fi
+
+			BPF_KERNEL_SOURCES_URL="http://mirrors.edge.kernel.org/pub/linux/kernel/v${kernel_version_major}.x/linux-${kernel_version}.tar.gz"
+		fi
+
+		if [ -n "${SYSDIG_BPF_USE_LOCAL_KERNEL_SOURCES}" ]; then
```
Comment (nathan-b) [inline scripts/falco-probe-loader]: SYSDIG -> FALCO

File: scripts/falco-probe-loader [hunk 5]
```diff
@@ -0,0 +1,408 @@
...
+	else
+		echo "* Failure to find a BPF probe"
+		exit 1
+	fi
+}
+
+ARCH=$(uname -m)
+KERNEL_RELEASE=$(uname -r)
+SCRIPT_NAME=$(basename "${0}")
+SYSDIG_PROBE_URL=${SYSDIG_PROBE_URL:-https://s3.amazonaws.com/download.draios.com}
```
Comment (nathan-b) [inline scripts/falco-probe-loader]: SYSDIG -> FALCO

File: scripts/falco-probe-loader [hunk 6]
```diff
@@ -0,0 +1,408 @@
...
+else
+	FALCO_PROBE_CURL_OPTIONS=-fsS
+fi
+
+MAX_RMMOD_WAIT=60
+if [[ $# -ge 1 ]]; then
+	MAX_RMMOD_WAIT=$1
+fi
+
+if [ -z "${SYSDIG_REPOSITORY}" ]; then
```
Comment (nathan-b) [inline scripts/falco-probe-loader]: SYSDIG -> FALCO

File: scripts/falco-probe-loader [hunk 7]
```diff
@@ -0,0 +1,408 @@
...
+	echo "Installer must be run as root (or with sudo)."
+	exit 1
+fi
+
+if ! hash curl > /dev/null 2>&1; then
+	echo "This program requires curl"
+	exit 1
+fi
+
+if [ -v SYSDIG_BPF_PROBE ] || [ "${1}" = "bpf" ]; then
```
Comment (nathan-b) [inline scripts/falco-probe-loader]: SYSDIG -> FALCO

File: docker/dev/docker-entrypoint.sh (hunk 1 above)
Comment (leodido) [reply docker/dev/docker-entrypoint.sh]: Yes in the plan to change it. Going to do it now

**Comment by leodido** [comment]
Closed in favor of #922 for using the convention of testing the branches with the same name together.

**Pull request closed without merging**